allennlp/training/trainer.py

```diff
@@ -166,6 +166,7 @@ def __init__(self,
                  patience: Optional[int] = None,
                  validation_metric: str = "-loss",
                  validation_iterator: DataIterator = None,
+                 shuffle: bool = True,
```

Comment:
`Trainer` has a custom from_params, so you'll need to add it there too

Reply:
good catch, thanks!